clang-tools-extra/test/clang-tidy/esri-no-implementation-in-headers.hpp

```diff
@@ -191,3 +191,26 @@ class StaticConstExpr {
     return 2;
   }
 };
+
+constexpr int foo = 42; // OK: constexpr variable defined in header.
+
+template<typename T>
+struct EB {
```

Comment:
Can this have a more contextual name. It is unfortunate that the test cases above didn't but would be good to circle back.

Reply:
Sounds Good. I think that's the way to go if sticking to the earlier pattern is not an issue.